### PR TITLE
release-checklist: update packaging instructions

### PR DESCRIPTION
**Pull request by arithx**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -5,26 +5,43 @@ Release checklist:
...
+   - [ ] Update the script and test it locally by dropping the `sigul` lines.
+ - [ ] Ping `mboddu` in Freenode `#fedora-coreos`, linking to the ticket
+ - [ ] Wait for the ticket to be closed
+ - [ ] Download the artifacts and signatures
+ - [ ] Verify the signatures
+ - [ ] Create a [draft release](https://github.com/coreos/fcct/releases/new) on GitHub and upload all the release artifacts and their signatures. Copy and paste the release notes from NEWS here as well.
  - [ ] Publish the release
+
+Quay release:
  - Update the `release` tag on Quay:
```
Comment (bgilbert) [inline .github/ISSUE_TEMPLATE/release-checklist.md:44]: Drop this line.

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -5,26 +5,43 @@ Release checklist:
  - [ ] Ensure you can sign commits and any yubikeys/smartcards are plugged in
```
Comment (bgilbert) [inline .github/ISSUE_TEMPLATE/release-checklist.md:5]: Add `Tagging:` section after `Release checklist:`.

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -5,26 +5,43 @@ Release checklist:
...
+Packaging:
+ - [ ] Update the FCCT spec file in [Fedora](https://src.fedoraproject.org/rpms/fedora-coreos-config-transpiler):
+   - Update the commit hash global variable
+   - Bump the `Version`
+   - Switch the `Release` back to `1.git%{shortcommit}%{?dist}`
+   - Remove any patches obsoleted by the new release
+   - Run `go-mods-to-bundled-provides.py | sort` while inside of the fcct directory you ran `./tag_release` from & copy output into spec file in `# Main package provides` section
+   - Add any new spec paths to `%gotest` lines
+   - Update change log
+ - [ ] Download `fcct-$VERSION.tar.gz` from the GitHub tag
```
Comment (bgilbert) [inline .github/ISSUE_TEMPLATE/release-checklist.md:18]: As in https://github.com/coreos/ignition/pull/1119, let's use `spectool` here too: "Run `spectool -g -S ignition.spec`"

**Review by bgilbert — COMMENTED** [review]
LGTM generally.  Added some comments to reduce the diff with the Ignition checklist.

**Comment by arithx** [comment]
Updated

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 4]
```diff
@@ -1,30 +1,48 @@
...
- - [ ] Sign the release artifacts by running
-```
-gpg --local-user 0xCDDE268EBB729EC7! --detach-sign --armor <path to artifact>
-```
-for each release artifact. Do not try to sign all of them at once by globbing. If you do, gpg will sign the combination of all the release artifacts instead of each one individually.
 
- - [ ] Create a draft release on GitHub and upload all the release artifacts and their signatures. Copy and paste the release notes from NEWS here as well.
+Packaging:
+ - [ ] Update the FCCT spec file in [Fedora](https://src.fedoraproject.org/rpms/fedora-coreos-config-transpiler):
+   - Update the commit hash global variable
```
Comment (bgilbert) [inline .github/ISSUE_TEMPLATE/release-checklist.md:13]: We don't have one of those in FCCT.

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 5]
```diff
@@ -1,30 +1,48 @@
...
+   - Bump the `Version`
+   - Switch the `Release` back to `1.git%{shortcommit}%{?dist}`
+   - Remove any patches obsoleted by the new release
+   - Run `go-mods-to-bundled-provides.py | sort` while inside of the fcct directory you ran `./tag_release` from & copy output into spec file in `# Main package provides` section
+   - Add any new spec paths to `%gotest` lines
```
Comment (bgilbert) [inline .github/ISSUE_TEMPLATE/release-checklist.md:18]: Nor this.

**Comment by arithx** [comment]
updated

**Review by bgilbert — APPROVED** [review]

**Pull request merged**